docs/de/docs/tutorial/middleware.md

```diff
@@ -0,0 +1,61 @@
+# Middleware
+
+Sie können Middleware zu **FastAPI**-Anwendungen hinzufügen.
+
+Eine "Middleware" ist eine Funktion, die mit jedem **Request** arbeitet, bevor er von einer bestimmten *Pfadoperation* verarbeitet wird. Und auch mit jeder **Response**, bevor sie zurückgegeben wird.
+
+* Sie nimmt jeden **Request** entgegen, der an Ihre Anwendung gesendet wird.
+* Sie kann dann etwas mit diesem **Request** tun oder beliebigen Code ausführen.
+* Dann gibt sie den **Request** zur Verarbeitung durch den Rest der Anwendung weiter (durch eine bestimmte *Pfadoperation*).
+* Sie nimmt dann die **Response** entgegen, die von der Anwendung generiert wurde (durch eine bestimmte *Pfadoperation*).
+* Sie kann etwas mit dieser **Response** tun oder beliebigen Code ausführen.
+* Dann gibt sie die **Response** zurück.
+
+!!! note "Technische Details"
+    Wenn Sie Abhängigkeiten mit `yield` haben, wird der Exit-Code *nach* der Middleware ausgeführt.
+
+    Wenn es Hintergrundaufgaben gab (später dokumentiert), werden sie *nach* allen Middlewares ausgeführt.
+
+## Erstellung einer Middleware
+
+Um eine Middleware zu erstellen, verwenden Sie den Dekorator `@app.middleware("http")` über einer Funktion.
+
+Die Middleware-Funktion erhält:
+
+* Den `request`.
+* Eine Funktion `call_next`, die den `request` als Parameter erhält.
+    * Diese Funktion gibt den `request` an die entsprechende *Pfadoperation* weiter.
+    * Dann gibt es die von der entsprechenden *Pfadoperation* generierte `response` zurück.
+* Sie können die `response` dann weiter modifizieren, bevor Sie sie zurückgeben.
+
+```Python hl_lines="8-9  11  14"
+{!../../../docs_src/middleware/tutorial001.py!}
+```
+
+!!! Tipp
```

Comment:
Sollte so sein (wegen der Farbe der Infobox):

```
!!! tip "Tipp"
```

Ist in [meinem .md](https://github.com/tiangolo/fastapi/pull/10391#issuecomment-1760536334) auch falsch, übersehen.